doc/Rules.md

```diff
@@ -1601,3 +1601,41 @@ To fix the violation, add a newline character to the end of the file:
 This file ends with a newline.
 [EOF]
 ```
+
+<a name="md048"></a>
+
+## MD048 - Code fence style
+
+Tags: code
+
+Aliases: code-fence-style
+
+Parameters: style ("consistent", "tilde", "backtick"; default "consistent")
+
+This rule is triggered when the symbols used in the document for fenced code blocks do not match the configured code fence style:
+
+```markdown
+```ruby
+# Fenced code
+```
+
+~~~ruby
+# Fenced code
+~~~
+```
```

Comment:
This does not render correctly because of the nested fences: https://dlaa.me/markdownlint/#%25m%23%20Demo%0A%0A%60%60%60markdown%0A%60%60%60ruby%0A%23%20Fenced%20code%0A%60%60%60%0A%0A~~~ruby%0A%23%20Fenced%20code%0A~~~%0A%60%60%60%0A

Using 4 backticks for the outer fences will fix the issue here and in the next example below.

Reply:
Uh, markdown syntax error in a markdown linter project. Thanks for the catch!

Reply:
I used 4 backticks now